static/src/javascripts/projects/common/modules/identity/wizard.js

```diff
@@ -243,17 +288,19 @@ export const enhance = (wizardEl: HTMLElement): Promise<void> =>
 
         wizardEl.addEventListener('click', (ev: Event) => {
             if (
-                ev.target instanceof HTMLElement &&
-                ev.target.closest(`.${nextButtonElClassname}`)
+                ev.target.closest &&
+                ev.target.closest instanceof Function &&
```

Comment:
`closest` will always be a function, no?

Reply:
flow doesn't recognize `.closest()` in `EventTarget` and i ran out of ideas to make it work. Is there a better way? On a fresh mind i think casting `EventTarget` to `HTMLElement` might work?

Reply:
Hmm, annoying. I'm surprised `ev.target instanceof HTMLElement` (or maybe `ev.target instanceof Element`) no longer works, since a [few people](https://github.com/facebook/flow/issues/218#issuecomment-74119319) have had some luck with it. I wonder what has changed?

If the above doesn't work, let's just leave it like this.

A more long-term solution would be to create a [custom `Event` class](https://github.com/facebook/flow/issues/218#issuecomment-255379074) that implements `target`, but that's something we can pick up later and needn't be tied to this change.

Reply:
AH i remember now (added a comment), checking for `HTMLElement` excluded elements like svgs & canvases which made the button not click sometimes

Reply:
Frustrating that in order to support types that only exist at compile time, we have to make checks that affect (and in this instance break) runtime execution.

Reply:
there's probably cleaner ways to get around this, i'm a flow newbie after all :(